kornia/augmentation/container/video.py

```diff
@@ -22,6 +25,12 @@ class VideoSequential(ImageSequential):
         *args (_AugmentationBase): a list of augmentation module.
         data_format (str): only BCTHW and BTCHW are supported. Default: BTCHW.
         same_on_frame (bool): apply the same transformation across the channel per frame. Default: True.
+        random_apply(int, (int, int), optional): randomly select a sublist (order agnostic) of args to
```

Comment:
maybe not for this PR, but as soon that all containers have the `random_apply` feature why not implementing it at base level ?

Reply:
The base level of containers is the ```ImageSequential``` class. This is implemented there but still needs some tweaks in different scenarios.